setup.py

```diff
@@ -0,0 +1,300 @@
+"""Setup script.
```

Comment:
@italovalcy setup.py looks good to me, but just realized that the [`apscheduler` dep is not being installed as is being used here](https://github.com/amlight/scheduler/blob/master/main.py#L9), could you add it on `requirements/run.in`? I was going to suggest the commit but for empty file it doesn't give the option.

Reply:
Addresses in commit 0622782